backend/django_core/static_src/components/specific/LatestEpisode/Index.svelte

```diff
@@ -0,0 +1,133 @@
+<script lang="ts">
+    import ScrollArea from "$components/minor/ScrollArea/Index.svelte";
+    import { FormatDate } from "$functions/format_date";
+    import Play from "$icons/Play/Index.svelte";
+    import { onMount } from "svelte";
+    import { slide } from "svelte/transition";
+
+    export let anime_id: number;
+    export let anime_cover: string;
+    export let anime_name: string;
+    export let anime_episode_number: number;
+    export let anime_release_date: string;
+    export let anime_synopsis: string;
+    export let anime_index: number;
+
+    // Formated anime details
+    const formated_episode_number = String(anime_episode_number).padStart(2, "0");
+    const formated_release_date = new FormatDate(anime_release_date).format_to_time_from_now;
+
+    /* Bindings */
+    let ANIMATION_DURATION = 300,
+        visible_ratio: number;
+    let scroll_area_element: HTMLElement, anime_episode: HTMLElement;
+    let show_more_info = false,
+        should_expand = false,
+        first_time_open = anime_index == 0;
+
+    onMount(() => {
+        // needed to drill two more layer cause of svelte-retag
+        scroll_area_element = anime_episode?.parentElement?.parentElement?.parentElement?.parentElement!;
```

Comment:
How's this, I wanted 2 more drilling :")

Reply:
I dont like it at all, find a better way to do this. maybe do svelte compoennt parent